packages/kbn-monaco/src/esql/lib/ast/ast_walker.ts

```diff
@@ -277,7 +277,7 @@ function visitOperatorExpression(
   if (ctx instanceof ArithmeticUnaryContext) {
     const arg = visitOperatorExpression(ctx.operatorExpression());
     // this is a number sign thing
-    const fn = createFunction('multiply', ctx);
+    const fn = createFunction('*', ctx);
```

Comment:
@dej611 can you explain this change?

Reply:
Builtin math function used to be called with their long names initially, but soon after I've changed them to use the actual symbol as name too.
Due to its lack of tests in this area I've never noticed the left over here which I'm fixing in this PR.